UM/PackageManager.py

```diff
@@ -402,3 +404,51 @@ def getPackageLicense(self, filename: str) -> Optional[str]:
                                             file_info.filename)
                         license_string = None
         return license_string
+
+    ##  Find the package files by package_id by looking at the installed folder
+    @staticmethod
+    def getPackageFiles(package_id) -> List[Tuple[str, List[str]]]:
+        data_storage_dir = os.path.abspath(Resources.getDataStoragePath())
+
+        os_walk = []
+        dirs_to_check = []
+        result = []  # 2-tuples of (dir, file_names)
+        for root_path, dir_names, file_names in os.walk(data_storage_dir):
```

Comment:
Can be errors if `data_storage_dir` is `None`

Reply:
data_storage_dir is the result of os.path.abspath and that's always a string.
os.path.abspath's input is Resources.getDataStoragePath() and that also always returns a string.